test/e2e/magellan-jetpack.json

```diff
@@ -1,5 +1,5 @@
 {
-	"mocha_tests": [ "specs/specs-calypso", "specs/specs-jetpack" ],
+	"mocha_tests": [ "specs/jetpack" ],
```

Comment:
This is a tangential point, but frankly I'm not sure of the value in keeping these Jetpack tests alive, when they:
1. hadn't been run in _many_ months;
2. are permafailing even if run;
3. no attention has been given in months, potentially years;

I think it might be time to retire the Jetpack build configuration on CI and associated configuration files in the repo, plus the specs themselves.

Reply:
>I think it might be time to retire the Jetpack build configuration on CI and associated configuration files in the repo, plus the specs themselves.

100% agreed! You alright if we do this on a separate PR? Makes the git trail cleaner and we could make sure the right stakeholders know!